### PR TITLE
MNT: make loggers behave more quietly

### LINKED ISSUE #190: Investigate spammy PLY log again
```
INFO     Loading daq...
SUCCESS  Successfully loaded daq
INFO     Loading scan_pvs (disabled)...
SUCCESS  Successfully loaded scan_pvs (disabled)
INFO     Loading elog...
SUCCESS  Successfully loaded elog
INFO     Loading database...
INFO     Created by PLY version 3.11 (http://www.dabeaz.com/ply)
INFO     Parsing method: LALR
INFO
INFO     state 0
INFO
INFO         (0) S' -> . nodelist
INFO         (8) nodelist -> . node
INFO         (9) nodelist -> . node nodelist
INFO         (10) nodelist -> .
INFO         (11) node -> . block
INFO         (12) node -> . command
INFO         (13) node -> . CODE
INFO         (14) node -> . COMMENT
INFO         (15) block -> . BARE ( arglist )
INFO         (16) block -> . BARE ( arglist ) { nodelist }
INFO         (17) command -> . BARE BARE
INFO         (18) command -> . BARE QUOTED
INFO
INFO         $end            reduce using rule 10 (nodelist -> .)
INFO         CODE            shift and go to state 5
INFO         COMMENT         shift and go to state 6
```

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

Quiet down various loggers that have decided to make INFO their personal high-priority debug streams.
closes #190 

## Motivation and Context

Super spammy on startup and during bluesky scans, and always if you imported any ophyd sim devices...

I will bring this into the pcdsutils soon (probably)

### CODE REVIEW

**Comment by ZLLentz** [comment]
Some more clarification about `ophyd` and `bluesky`:
https://github.com/bluesky/ophyd/blob/eee8797e79448a1dc02c65952af475563f4d260a/ophyd/signal.py#L214-L217
https://github.com/bluesky/bluesky/blob/fdb879763a0fd899474af7a4406736280050518b/bluesky/run_engine.py#L781

We're also stuck on an older version of `bluesky` that configures its own logger automatically until I give up and add the nsls-ii channels to our conda envs or until I decide to become a maintainer of the `conda-forge` feedstock for `bluesky`

File: hutch_python/log_setup.py [hunk 1]
```diff
@@ -57,10 +57,20 @@ def setup_logging(dir_logs=None):
...
+def hush_noisy_loggers(modules, level=logging.WARNING):
+    """
+    Some loggers spam on INFO with no restraint, so we must raise their levels.
+
+    It seems there is some disagreement over what log levels should mean. In
+    our repos, INFO is used as the de-facto print replacement, but in some
+    repos it is used as the secondary debug stream.
+    """
+    for module in modules:
+        logging.getLogger(module).setLevel(logging.WARNING)
```
Comment (ZryletTC) [inline hutch_python/log_setup.py:73]:
```suggestion
        logging.getLogger(module).setLevel(level)
```

Comment (ZLLentz) [reply hutch_python/log_setup.py:73]: oof

File: hutch_python/log_setup.py [hunk 2]
```diff
@@ -57,10 +57,20 @@ def setup_logging(dir_logs=None):
...
     logging.config.dictConfig(config)
-    # Disable parso logging because it spams DEBUG messages
-    # https://github.com/ipython/ipython/issues/10946
-    logging.getLogger('parso.python.diff').disabled = True
-    logging.getLogger('parso.cache').disabled = True
+    noisy_loggers = ['parso', 'pyPDB.dbd.yacc', 'ophyd', 'bluesky']
+    hush_noisy_loggers(noisy_loggers)
+
+
+def hush_noisy_loggers(modules, level=logging.WARNING):
```
Comment (klauer) [inline hutch_python/log_setup.py:64]: 🅰️ ➕  function name 😆

**Pull request merged**